src/viam/sdk/rpc/dial.hpp

```diff
@@ -16,9 +16,15 @@ class ViamChannel {
    public:
     void close();
     ViamChannel(std::shared_ptr<GrpcChannel> channel, const char* path, void* runtime);
+
     static std::shared_ptr<ViamChannel> dial(const char* uri,
                                              const boost::optional<DialOptions>& options);
 
+    // Specifies that this dial is an initial connection attempt and that initial connection options
```

Comment:
This seems as good a time as any to doc comment this function, as well as `dial` above.

Reply:
Done; added a clause as well clarifying that use of `dial` is discouraged and that `RobotClient::at_address(...)` is preferred.